config/metadata/configMeta.yaml

```diff
@@ -1258,6 +1258,7 @@ groups:
           `MB` and `GiB`) and Kubernetes units (such as `M` and `Gi`) are
           supported. Fractional values with a suffix are supported. If
           `AvailableMemory` is set, `Collections.MaxAlloc` must not be defined.
+          A useful value for this setting is about 85% of the pod's total memory.
```

Comment:
It would be great to have more information re. the 85% recommendation. "about 85%" makes me wonder why not 90, or 80

Reply:
Maybe something like "A useful value is to leave 1-2GB of pod memory for overages. For typical configurations, this might be 85 to 90% of the pod's total memory."

Reply:
Used your wording @kentquirk